packages/cli/src/cmds/agentInstaller/javaBuildToolInstaller.ts

```diff
@@ -5,29 +5,8 @@ import chalk from 'chalk';
 import CommandStruct from './commandStruct';
 import { run } from './commandRunner';
 import { getOutput } from './commandUtil';
-import { findIntelliJHome } from './jetBrainsSupport';
 import AgentInstaller from './agentInstaller';
 
-export function addJetBrainsEnv() {
-  const jbHome = findIntelliJHome();
-  if (!jbHome) {
-    return;
-  }
-
-  const javaHome = path.join(jbHome, 'jbr');
-  const mvnBin = path.join(jbHome, 'plugins/maven/lib/maven3/bin');
-
-  // Make sure we don't override the user's settings: append to path, use the
-  // existing JAVA_HOME if it's set.
-  if (!process.env['JAVA_HOME']) {
-    process.env['JAVA_HOME'] = javaHome;
-  }
```

Comment:
It's not clear to me how this could have overwritten JAVA_HOME. It seems pretty explicitly trying not to do that.